### ISSUE TITLE
[Suggestion] Add updating of user emails, usernames and passwords

### ISSUE THREAD

**Issue by Hutch79** [Lynx-Shortener/Lynx#86]
**Describe the solution you'd like**
- User Deletion
- User Editing (Mail/Username changes and Password resets)

More ideas/suggestions which are more advance and not as important as the first two:
- Self Password Reset over Mail
- URL Limits - Limit a user to e.g. 10 URLs

**Describe alternatives you've considered**
I don't know if there is any official CLI for Lynx.

**Additional context**
User deletion as well as a possibility to edit a User as Admin are the most important features.
Everything else is a nice to have.

With the 1.7 Update, Lynx got quite a huge improvement, which is really handy if you want to give friends and colleagues access to Lynx.
In other words, my favourite URL shortener just got better!
Now it only lacks some administrative stuff to complete the package.
Thanks for keep working on Lynx, I really enjoy this software, and it's probably one of the small things I use most in my docker stack.

**Comment by Hutch79** [comment]
> You can delete other users in the user management page if their role is lower than yours. I can add self-deletion if it's wanted.
Heya, thanks for the fast response!
Unfortunately, I can't see the option, although I'm the Owner.
![image](https://github.com/Lynx-Shortener/Lynx/assets/42042811/d868d2d0-deef-4cda-bcfd-a35fa1508298)

**Comment by JackBailey** [comment]
Hm there's meant to be an icon on the right of the secret box. I'll take a look

**Comment by Hutch79** [comment]
Hmm, I just updated to 1.7.2.
I needed to reinstall everything because my account somehow wasn't listed as Owner after the update (only had a handful of links).
But I updated from 1.5.4-beta if I'm not mistaken.

**Comment by JackBailey** [comment]
Being added in 1.7.3, which I will release soon.
I'll add promoting users to owner if one doesn't exist too for upgrading from older versions.

**Comment by JackBailey** [comment]
Other changes will be added in 1.8.0

**Comment by Hutch79** [comment]
> URL limits and user detail editing will be added in 1.8.0.
Password resets will be added later, potentially 1.9.0.

Nice, thank you very much!

The CLI was just a tough of me if it may be possible to do it over a CLI because it wasn't in the UI, but I personally wouldn't use a CLI very often ^^'

**Comment by JackBailey** [comment]
I've added updating of username, email and password. It'll be released in [1.8.0](https://github.com/Lynx-Shortener/Lynx/milestone/3)
